### ISSUE TITLE
[Tutorial 9] The headers module is loaded twice

### ISSUE THREAD

**Issue by studersi** [dune73/test-4#55]
The module `headers_module` is loaded twice and Apache issues a warning.

LoadModule        mpm_event_module        modules/mod_mpm_event.so
LoadModule        unixd_module            modules/mod_unixd.so

LoadModule        log_config_module       modules/mod_log_config.so
LoadModule        logio_module            modules/mod_logio.so
LoadModule        rewrite_module          modules/mod_rewrite.so
**LoadModule        headers_module          modules/mod_headers.so**

LoadModule        authn_core_module       modules/mod_authn_core.so
LoadModule        authz_core_module       modules/mod_authz_core.so

LoadModule        ssl_module              modules/mod_ssl.so
**LoadModule        headers_module          modules/mod_headers.so**

LoadModule        unique_id_module        modules/mod_unique_id.so
LoadModule        security2_module        modules/mod_security2.so

LoadModule        proxy_module            modules/mod_proxy.so
LoadModule        proxy_http_module       modules/mod_proxy_http.so
LoadModule        proxy_balancer_module   modules/mod_proxy_balancer.so
LoadModule        lbmethod_byrequests_module modules/mod_lbmethod_byrequests.so
LoadModule        slotmem_shm_module      modules/mod_slotmem_shm.so

**Comment by dune73** [comment]
Yes, it's actually being introduced in tutorial-4 and used throughout up to tutorial-9. 

The 2nd addition here under mod_rewrite is a simple flaw. Would you mind doing a PR for this.
